hoot-core/src/main/cpp/hoot/core/algorithms/changeset/ChangesetCreator.cpp

```diff
@@ -422,7 +422,7 @@ void ChangesetCreator::_handleUnstreamableConvertOpsInMemory(
   convertOps.setProgress(
     Progress(
       ConfigOptions().getJobId(), JOB_SOURCE, Progress::JobState::Running,
-      (float)(_currentTaskNum - 1) / (float)_numTotalTasks, 1.0 / (float)_numTotalTasks));
+      (_currentTaskNum - 1) / _numTotalTasks, 1.0 / _numTotalTasks));
```

Comment:
The problem that is flagged here is that `1.0` is actually a `double` and not a `float`.  The correct way to fix this is to do the following:
```
(float)(_currentTaskNum - 1) / (float)_numTotalTasks, 1.0f / (float)_numTotalTasks));
```